Do not attempt to use MacPorts OpenMP.
Do not install libraries a second time.
--- CMakeLists.txt.orig	2019-02-06 05:24:33.000000000 -0700
+++ CMakeLists.txt	2019-02-15 05:23:25.000000000 -0700
@@ -292,14 +292,6 @@
     set_config_option(HAVE_OPENMP "OpenMP")
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${OpenMP_C_FLAGS}")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
-  elseif(APPLE AND EXISTS "/opt/local/lib/libomp")
-    # just for testing the official Apple compiler with macports' libomp
-    set_config_option(HAVE_OPENMP "OpenMP[MacPorts]")
-    set(CMAKE_C_FLAGS
-        "${CMAKE_C_FLAGS} -Xpreprocessor -fopenmp -I/opt/local/include/libomp")
-    set(CMAKE_CXX_FLAGS
-        "${CMAKE_CXX_FLAGS} -Xpreprocessor -fopenmp -I/opt/local/include/libomp")
-    list(APPEND EXTERNAL_LIBRARIES "-L/opt/local/lib/libomp -lomp")
   endif()
 endif()
 
@@ -1817,8 +1809,6 @@
 endif()
 if(ENABLE_BUILD_SHARED OR ENABLE_BUILD_DYNAMIC)
   install(TARGETS shared DESTINATION ${GMSH_LIB} OPTIONAL)
-  # FIXME once we cleanup the installation of the python module
-  install(TARGETS shared DESTINATION ${CMAKE_CURRENT_SOURCE_DIR}/api OPTIONAL)
 endif()
 
 if(ENABLE_ONELAB)
